src/content/docs/new-relic-control/agent-control/configuration.mdx

```diff
@@ -0,0 +1,284 @@
+</Callout>
+
+The [`values-newrelic.yaml`](https://github.com/newrelic/helm-charts/blob/master/charts/agent-control/values.yaml) file, which traditionally defined agent behavior, now also includes settings for Agent Control. The settings and parameters that you configure in this file define how Agent Control and the agents it manages operate. This file is referred to as the local configuration.
+
+Here's an example configuration:
+
+<CollapserGroup>
+  <Collapser
+    id="agent-control-config"
+    title="Agent Control configuration"
+  >
+
+    ```yaml
+    global:
+      cluster: "YOUR_CLUSTER_NAME"
+      licenseKey: "YOUR_LICENSE_KEY"
+      userKey: "YOUR_USER_KEY"
+
+    # Values related to the agent control's Helm chart release.
+    # `https://github.com/newrelic/helm-charts/blob/master/charts/agent-control/values.yaml`
+    agent-control-deployment:
+      config:
+        fleet_control:
+          # Optional: Specify a fleet_id (entity guid) to automatically connect to an existing fleet.
+          fleet_id: ""
+          auth:
+            # New Relic organization ID where agent will connect to.
+            organizationId: "YOUR_ORGANIZATION_ID"
+        
+        # List of managed agents that will be deployed. The key represents the name of the agent and the value holds the configuration.
+        subAgents:
+          infrastructure:
+            type: newrelic/com.newrelic.infrastructure:0.1.0
+            content:
+              # Ref: `https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle`
+              # Recommended: check and define an explicit chart version (latest stable)
+              chart_version: "*"   
+              chart_values:
+                newrelic-infrastructure:
+                enableProcessMetrics: true
+          logs:
+            type: newrelic/io.fluentbit:0.1.0
+            content:
+              # Ref: `https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging`
+              # Recommended: check and define an explicit chart version (latest stable)
+              chart_version: "*"
+              chart_values:
+                newrelic-logging:
+                  sendMetrics: true
+          agent-operator:
+            type: com.newrelic.k8s_agent_operator:0.1.0
+            content:
+              chart_version: "*"
+    ```
+
+  </Collapser>
+
+</CollapserGroup>
+
+The example shows the Agent Control configuration, which includes the settings for the Kubernetes infrastructure agent as well as for Fluent Bit.
+
+For environments with a small number of clusters, managing local configurations is straightforward. For example, if you don't want to send metrics related to the health of your log collector (Fluent Bit, in this case), set `sendMetrics` to `false` in the YAML file, and run the installation command.
```

Comment:
I do not understand why explaining a low level detail of a single agent here.
I understand that `sendMetrics` is there by chance as an example